### ISSUE TITLE
Code works but won't validate.

### ISSUE THREAD

**Issue by ccsears** [FreeCodeCamp/FreeCodeCamp#3487]
Challenge [Bonfire: Seek and Destroy](http://freecodecamp.com/challenges/bonfire-seek-and-destroy) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; rv:40.0) Gecko/20100101 Firefox/40.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```
function destroyer(arr) {
  // Remove all the values

 var arr = arguments[0];
 var z = [];
 var j = 0;
 for (i = 1; i < arguments.length-1; i++) {
    z[j]=arguments[i];
    j = j+1;
  }
  
  var chris = 0;
  var leny = arr.length;
  for (i = 0; i < z.length; i++) {
   for (k = 0; k < z.length; k++) {
     chris = arr.indexOf(z[k]);
     if (chris != -1) {
       arr.splice(chris, 1);
     }
   }
  }
  
  return arr;
}


destroyer(["tree", "hamburger", 53], "tree", 53, "");

```

**Comment by ccsears** [comment]
In the display I get the correct answer but I don't pass any of the tests.

**Comment by jhealy86** [comment]
It appears that your code is passing the default test given, but not any of the other test cases. You may have to take a look at your code and find a way to get it to work. If you need further help with this you may go to the chat channels, especially the [Default channel](https://gitter.im/freecodecamp/FreeCodeCamp) and the [HelpBonfires channel](https://gitter.im/freecodecamp/HelpBonfires).

**Comment by awesomeaniruddh** [comment]
Thanks for reporting the issue @ccsears, but as @jhealy86 said, your code does not pass the tests. Closing this.

**Comment by ccsears** [comment]
It passes all the test listed.  I tried every test case and they ALL came back with a red "X".  However in the black display box it had the CORRECT answer.  Did you even run my solution?

**Comment by jhealy86** [comment]
Yes, I did run your code. I know exactly what's going on with it and why it's failing, but this is not the place for that. If you stop by the help channels I posted earlier there are people (myself included) who would be more than happy to help you out to find the problem in your code. But this is not the place for that. The Bonfire itself is working correctly in this case. The one hint I will give you is to look carefully at the default test case given in the code and the first one being run on the left side. There's a subtle difference, but that's what you're getting tripped up on.
